pommerman/agents/player_agent.py

```diff
@@ -1,13 +1,70 @@
+# NOTE:
```

Comment:
I like all of this documentation. Please wrap it in a comment string instead of "#"